### ISSUE TITLE
Invalid formatting using the inline let pattern

### ISSUE THREAD

**Issue by xavierjurado** [nicklockwood/SwiftFormat#176]
The `inline` let pattern produced non-compilable code under the following circumstances:

Options:

> swiftformat --indent 4 --allman false --wraparguments beforefirst --wrapelements beforefirst --removelines enabled --exponentcase lowercase --hexliteralcase uppercase --binarygrouping none --decimalgrouping none --octalgrouping none --hexgrouping none --stripunusedargs closure-only --self remove --header ignore --insertlines enabled --empty void --ranges nospace --trimwhitespace always --linebreaks lf --commas always --comments indent --ifdef indent --patternlet inline --semicolons inline --disable redundantRawValues

Input code:

```swift
switch error {
case let (ApplicationError.invalidCredentials(message)):
}
```

Formatted code (note the presence of two `let` statements):

```swift
switch error {
case (let FotocasaError.invalidCredentials(let message)):
}
```

**Comment by nicklockwood** [comment]
Thanks. I've found the bug - will be fixed in the next release.

**Comment by nicklockwood** [comment]
In the meantime, you can use `--disable hoistPatternLet` as a workaround.

**Comment by nicklockwood** [comment]
@xavierjurado fixed in 0.29.6. Thanks for reporting!
